src/routes/LoginPage.vue

```diff
@@ -39,6 +39,10 @@
           </v-btn>
         </v-container>
       </v-form>
+      <a
```

Comment:
Can you use `router-link` instead of traditional `a` link ? The router link allow to avoid a page reloading.
Usage example :
```vue
      <router-link
        to="/"
        class="home-link"
      >
        {{ $t('upsilon-workshop') }}
      </router-link>
```

Reply:
i will do that

Reply:
Change done